Formula/ats2-postiats.rb

```diff
@@ -1,8 +1,8 @@
 class Ats2Postiats < Formula
-  desc "ATS programming language implementation"
+  desc "ATS2: Unleashing the Potentials of Types and Templates"
```

Comment:
Thanks for your input, I'll tweak it to something along the lines of what you suggested next time I push a version update